joblib/test/conftest.py

```diff
@@ -0,0 +1,6 @@
+from joblib.testing import fixture
+
+
+@fixture(scope='function')
```

Comment:
I think what this is is me, as a new pytest user, being a bit wary about pytest magic ... Leave it like this is fine.